pmoves/services/comfy-watcher/watcher.py

```
@@ -77,7 +77,7 @@ async def run() -> None:
                     if state["uploaded"].get(h):
                         continue
 
-                    key = datetime.datetime.utcnow().strftime("comfyui/%Y/%m/%d/") + fn
+                    key = datetime.datetime.now(timezone.utc).strftime("comfyui/%Y/%m/%d/") + fn
```

Ruff (0.14.10)

Undefined name `timezone`

(F821)